awake.lua

```diff
@@ -299,34 +299,52 @@ function init()
   params:add{type = "trigger", id = "reset", name = "reset",
     action = function() reset() end}
   
-  params:add_group("synth",6)
+  params:add_group("synth", 7)
```

Comment:
When Ezra added detune, it was supposed to be part of the synth> group but this wasn't changed, and so it landed in the top menu. This fixed it.